salt/engines/kujira_parser/kujira_event_parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""Salt engine responsible for catching all events from calamari and putting them in Redis.
+Salt engines are a new feature in 2015.8.0
+
+Example master configuration:\n
+extension_modules: srv/salt\n
+engines_dir: srv/salt/engines/kujira_parser\n
+engines:
+    \- kujira_event_parser
+"""
+
+# Import python libs
+from __future__ import absolute_import
+import sys
+import json
+import logging
+
+# Import salt lib
+import salt.utils.event
+
+# import local module
+from .redisDbLib import RedisHandler
```

Comment:
RedisHandler will module/package available from Python path.
Talk with Patryk how to use it.